src/app/routes/profile/containers/editprofile.js

```diff
@@ -86,21 +103,33 @@ export default class EditProfile extends React.Component {
                 }
             },
             submitHandler: function(form) {
-                // Todo: Save changes here using refs
+                // Todo: Save changes here (this.refs)
+                store.dispatch(push('settings/profile'));
             }.bind(this)
         };
     }
 
     photoHandler(event) {
         event.preventDefault();
-        // Update photo preview
         // Todo: Fix image rendering issue. Some images render rotated 90 degrees, due to image metadata
         var reader = new FileReader();
         reader.onload = function (e) {
             this.refs.image.src = e.target.result;
         }.bind(this)
-        if (this.refs.imageSelect.files.length == 1) {
-            reader.readAsDataURL(this.refs.imageSelect.files[0]);
+        if (this.refs.imageSelect.files.length == FILE_CONSTANTS.selected) {
+            var file = this.refs.imageSelect.files[FILE_CONSTANTS.index];
```

Comment:
perhaps we can come up with better names for the constant so that it's more intuitive.